docs/ops.html

```diff
@@ -3822,6 +3822,9 @@ <h3>Provisioning the KRaft controller quorum</h3>
 # The inter broker listener in brokers to allow KRaft controller send RPCs to brokers
 inter.broker.listener.name=PLAINTEXT
 
+# Maps listener names to security protocols. Please add the inter broker listener protocol mapping
+listener.security.protocol.map=PLAINTEXT:PLAINTEXT
```

Comment:
I was wondering if we would make clear that even the CONTROLLER listener should be here. But I found this in the Kafka doc related to the `listener.security.protocol.map` prop:

> Note that in KRaft a default mapping from the listener names defined by controller.listener.names to PLAINTEXT is assumed if no explicit mapping is provided and no other security protocol is in use.

So if not setting CONTROLLER:PLAINTEXT, it's the default.
Not sure if we should set it in the doc to make it clearer. Not a strong opinion but I would like to know what do you think about it.
I mean, this doc seems to assume you have everything using PLAINTEXT as security protocol. If not, following these steps doesn't work pretty well out of box.

Reply:
You're right. PR updated to add `CONTROLLER:PLAINTEXT`